### ISSUE TITLE
some question about junction map

### ISSUE THREAD

**Issue by iifeve** [lhwcv/mlsd_pytorch#14]
Hi there! Thanks for your excellent work, but I notice that the dimension of junction map is (h, w, 1), but the others are (c, h ,w), and in phase of loading label, it didn't change the dimension. I wonder should it be this way or there is something I missed?

> def gen_junction_and_line_mask(norm_lines, h = 256, w = 256):
    junction_map = np.zeros((h, w, 1), dtype=np.float32)
    line_map = np.zeros((h, w, 1), dtype=np.float32)

> def load_label(self, ann, do_aug):

>          label[14, :, :] = junction_map[0]
            label[15, :, :] = line_map[0]

**Comment by ngdyes** [comment]
Yeah,I found it wrong too.That makes line and junction segment effectless.
